projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -69,6 +27,11 @@ const Index = ( { criticalCss }: IndexProps ) => {
 	const [ isaState ] = useSingleModuleState( 'image_size_analysis' );
 	const [ imageCdn ] = useSingleModuleState( 'image_cdn' );
 
+	const regenerateCssAction = useRegenerateCriticalCssAction();
+	const requestRegenerateCriticalCss = () => {
+		regenerateCssAction.mutate();
+	};
```

Comment:
I used `Action` as a suffix because (for example) `setProviderCss` sounds like a function. I wanted to make it clear in my code that it's not. I agree that it's bulky, but I feel it's the right answer.

I originally made my `use` action creators return a function which called mutate directly to keep my code in components cleaner - but I didn't like that prevents the caller from choosing whether to call `mutate` or `mutateAsync`.

I think I stand by this naming for now. We can always choose to adapt later.